### DISCUSSION TITLE
MSCS%40Columbia/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#306] [Announcements]
# MSCS%40Columbia/

None

https://opencs.app/MSCS%40Columbia/

**Comment by fiersdrag** [comment]
https://docs.google.com/presentation/d/1AML5hko22h42KR6bLO38fYklMqvLCtIH/edit?usp=sharing&ouid=116730452879259903552&rtpof=true&sd=true

**Comment by RainJamesY** [comment]
24fall 中9 3.9/4 也被rej了，过于玄学
